site/en/docs/extensions/mv3/architecture-overview/index.md

```diff
@@ -2,370 +2,107 @@
+See [Handling events in the extension service worker][doc-sw] for more details. 
 
-You can open these pages using the web api [window.open()][mdn-window-open], the Chrome APIs
-[windows.create()][api-window-create], or [tabs.create()][api-create-tab].
+### Content scripts {: #content-scripts }
 
-## Extension files {: #files }
+Extensions use content scripts to inject code into host pages. They allow the extension to interact with and modify the pages in the browser. For example, they can insert a new element on the page, change the style of a website, modify the [DOM][mdn-dom] elements, etc. 
 
-### Referencing extension files {: #ref-files }
+Content Scripts share the same origin as their host page and they have access to a limited number of [Chrome APIs][api-ref]. See [Understanding content scripts][doc-content-scripts] for more details.
 
-Just as HTML pages on the web can include files on the same site with _relative URLs_, **extension
-pages** can also reference extension assets using relative paths.
+### Extension HTML pages {: #html-files }
 
-```html
-<img src="images/my_image.png">
-```
+An extension can have different HTML pages depending on the design. All extension HTML files can use the [Chrome APIs][api-ref], but cannot include inline Javascript; they must point to a JavaScript file. The two most common HTML pages are:
 
-To access an extension file from a **content script**, you can call
-[`chrome.runtime.getURL()`][api-get-url] to get the _absolute URL_ of your extension asset.
+[The popup][doc-popup]
+: Many extensions use a popup to provide functionality, such as displaying a list of tabs, or additional information regarding the current tab. Users can easily find it by clicking on the extension toolbar icon. When the user navigates away it will automatically close.
 
-``` js
-let image = chrome.runtime.getURL("images/my_image.png")
-```
+[The options page][doc-options]
+: The options page provides a way for users to customize an extension; such as choosing which sites the extension will run on. Users can access the options page in several ways as described in [Finding the options page][doc-options-view].
 
-To access an extension file from a **website**, you will have to construct the URL as follows:
+Other extension HTML pages include [Chrome override pages][doc-override], [sandbox pages][doc-sandbox] or any custom page included for a specific purpose like onboarding the user.
 
-```text
-chrome-extension://EXTENSION_ID/RELATIVE_PATH
-```
+### Other assets {: #assets }
 
-You can find the <code><var>EXTENSION_ID</var></code> in the Extension management page
-**chrome://extensions**. The <code><var>RELATIVE_PATH</var></code> is the file path relative to the
-extension's top folder.
+An extension can include many types of resources, but only the [extension icons][manifest-icons] are required. All assets, including images and fonts, must be part of the extension package.
```

Comment:
> All assets, including images and fonts, must be part of the extension package.

Is this a reference to remotely hosted code restrictions? If so, this isn't quite true. While remote scripts cannot be executed, the use of remotely hosted images, fonts, styles, configuration, and other non-code resources is acceptable.

Reply:
Thanks for the clarification! I removed this sentence.